.gitignore

```diff
@@ -1,2 +1,3 @@
 dist
 coverage
+node_modules
```

Comment:
Added this because it's a project-specific ignore, not OS specific. Feel free to remove it though.